tests/spread/core24/linters-file/task.yaml

```diff
@@ -1,4 +1,6 @@
 summary: Run snapcraft lint on a snap file.
+# Disabled from regular CI runs until we have "snapcraft lint" in core24
+manual: true
```

Comment:
We should add a note to the "snapcraft lint" task to re-enable this.

Reply:
this is done (on the Jira issue)